model/model.js

```diff
@@ -0,0 +1,14 @@
+const db = require ('../db/connection.js')
+
+exports.selectAllTopics = async () => {
+  try {
+
+    /* if (!colours.includes(colour)) {
+      return Promise.reject({status: 404, msg: 'Not found'})
```

Comment:
assume this commented out bit is from previous sprint. please don't copy and paste code!